fetchcode/github_tags.py

```diff
@@ -0,0 +1,46 @@
+# fetchcode is a free software tool from nexB Inc. and others.
+# Visit https://github.com/nexB/fetchcode for support and download.
+#
+# Copyright (c) nexB Inc. and others. All rights reserved.
+# http://nexb.com and http://aboutcode.org
+#
+# This software is licensed under the Apache License version 2.0.
+#
+# You may not use this software except in compliance with the License.
+# You may obtain a copy of the License at: 
+# http://apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software distributed
+# under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+# CONDITIONS OF ANY KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations under the License.
+
+import requests
+from urllib.parse import urlparse
+
+
+def build_url(base_url, path):
+    """
+    Builds URL according to GitHub API 
+    """
+    return base_url + path + '/tags'
```

Comment:
Can you describe what base_url  and path are ? what would examples of values? Also using + sounds rather britlle...
What about using format or a f string instead? and does base_url ever changes?

Reply:
And also is this function really warranted since you are not even testing it?

Reply:
I have tested the get_tags() function that is using this function so should I also write unit test for this function 😅